addons/binding/org.openhab.binding.wink/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="wink"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="wink_hub_2">
+    	<label>Wink Hub 2</label>
+	</bridge-type>
+	<thing-type id="light_bulb">
+        <supported-bridge-type-refs>
+        	<bridge-type-ref id="wink_hub_2" />
+        </supported-bridge-type-refs>
+        <label>Light Bulb (Dimmable)</label>
+        <description>Controls dimmable light bulb</description>
+
+        <channels>
+            <channel id="lightlevel" typeId="lightDimmer" />
+            <channel id="lightstate" typeId="lightSwitch" />
```

Comment:
Not sure I follow you here.  Light Level and Light state are separate channels.  If I set the light level to 50% and then set the state to off and then set the state back on again, the light comes back on at 50%.  Maybe I'm not understanding what you mean.

Reply:
What I mean is implemented in the [LifX Binding](https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/handler/LifxLightHandler.java#L332)

what is does is storing the information about the last dimmed state in a [channel configuration](https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.lifx/ESH-INF/thing/channel.xml#L14) as a [parameter](https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.lifx/ESH-INF/config/config.xml#L24).

There is no need for a separate ON/OFF channel.

Reply:
I've spent a few hours on this and I can't get this to work. I've added some code to save the brightness every time it gets updated, and to restore it when we turn on the device in PaperUI, but it seems like flipping the ON/OFF switch always update the slider to 100% by default, I can send an update to restore the appropriate slider's value but there's a big latency (the slider stays at 100% for a few seconds).

Is there a way to configure a Dimmer item so that it returns to its previous value when turned on, rather than to 100%?

Reply:
The "delay" in the slider comes due to the "auto update" feature, so a control in PaperUI/BasicUI/ClassicUI sends out the command and this command will be treated as the state of attached device (i.e. an optimistic estimation). In your case this is wrong but that is not a fault of your code.

There is a pending PR which enables you as a binding developer to disable this feature and then the "delay" will be gone, see https://github.com/eclipse/smarthome/pull/5011

So you can keep the rewritten code that stores the value in a channel configuration and later disable the auto update feature once the aforementioned PR is merged.

Reply:
Ha ok, that makes sense now :),  I thought that I was doing something wrong. I'll use the channel configuration approach.